coldfront/core/group/management/commands/create_default_user_groups.py

```diff
@@ -0,0 +1,190 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group, Permission
+
+# Default permissions for the RIS-UserSupport user_group
+DEFAULT_RIS_USERSUPPORT_GROUP_PROJECT_PERMISSIONS = [
+    {"id": 53, "codename": "add_project"},
```

Comment:
@gunnarwustl I am curious to know why the ids are specified instead of allowing the DB to set an ID. Perhaps, we want to keep uniform permissions records in every environment (qa, local, prod). Why was the sequence started at 54 or 167?

Reply:
I'm not creating these permissions. I'm assign existing permissions from our DB, based on their existing IDs, to this group I am creating. I don't specify the ID of the group upon creation as you may have noticed elsewhere.

@peaches7083 noted a preference to specifically set each permission here rather than to simply query the DB for "all allocation/project permissions" and then set those.

Reply:
Thanks